Src/Plugins/Security/SEC101_054.AlibabaAccessKeyValidator.cs

```diff
@@ -0,0 +1,119 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+// From the Nuget package
+using Aliyun.Acs.Core;
+using Aliyun.Acs.Core.Exceptions;
+using Aliyun.Acs.Core.Profile;
+
+// From the DLL
+using Aliyun.Acs.Iot.Model.V20170420;
+
+using Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security.Utilities;
+
+namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Plugins.Security
+{
+    public class AlibabaAccessKeyValidator : ValidatorBase
+    {
+        internal static AlibabaAccessKeyValidator Instance;
+
+        private const string _keyNotFound = "InvalidAccessKeyId.NotFound";
```

Comment:
>_keyNotFound [](start = 29, length = 12)

use 'KeyNotFound' as a name. We don't generally use the underscore prefix for a const member.

Reply:
Done

---
In reply to: [591881330](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591881330) [](ancestors = 591881330)